src/default/playing-flat-action-map.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I'd put this out from the core, in examples or somewhere else like templates or so